os/tools/mkbootparam.py

```diff
@@ -159,6 +160,14 @@ def make_bootparam():
 ###################################################
 # Generate boot parameters
 ###################################################
-FLASH_START_ADDR = sys.argv[1]
-FLASH_SIZE = sys.argv[2]
+# Check the board type.
+BOARD_TYPE = get_value_from_file(config_file_path, "CONFIG_ARCH_BOARD=").replace('"', '').rstrip("\n")
+metadata_file = build_folder + '/configs/' + BOARD_TYPE + '/board_metadata.txt'
+# Get the flash start address and its size from metadata file.
+if os.path.isfile(metadata_file) :
+	FLASH_START_ADDR = get_value_from_file(metadata_file, "FLASH_START_ADDR=").replace('"','').rstrip('\n')
+	FLASH_SIZE = get_value_from_file(metadata_file, "FLASH_SIZE=").replace('"','').rstrip('\n')
```

Comment:
How about adding replace('"','').rstrip('\n') to get_value_from_file function. These are being done everywhere after calling get_value_from_file.